flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/job/savepoints/stop/StopWithSavepointRequestBody.java

```diff
@@ -55,6 +55,6 @@ public String getTargetDirectory() {
 	}
 
 	public boolean shouldAdvanceToEndOfEventTime() {
```

Comment:
Maybe update the method name as well to `shouldDrain()` or sth along those lines.

Reply:
Done.